### PR TITLE
925387-Adding-UIKit-UG-Added Initial Files

### PR DESCRIPTION
**Pull request by HadayaRahman-SF4673**
Description:
Added .NET MAUI Essential UI Kit User Guide

### CODE REVIEW

File: MAUI/UIKit/GettingStarted.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+   ![Visual Studio Ui Kit Resources](UI-Kit-images/Essential_UIKit_ClassFiles.png)
+
+## How to render the added page
+
+In a MAUI demo application, you must make the added page as the start-up page in the AppShell.xaml file. 
+Example: If you added the Login with Social Icon Page, then you must invoke the page as demonstrated in the following code.
+
+1. Add the namespace as shown in the following code:
+{% highlight xaml %}
+<Shell
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:63]: Dont mention in shell content

**For NET 9 project:**
image

**For Net 8:**
image

File: MAUI/UIKit/maui-ui-designs.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+title: MAUI UI Design | Syncfusion
+description: Essential UI Kit contains elegantly designed XAML templates for MAUI apps. These templates are compatible with Android, iOS, and UWP platforms.
+platform: maui
+control: MAUI UI Kit
+documentation: ug
+---
+
+# Syncfusion MAUI UI Designs 
+
+The Essential UI Kit includes 101 screens which is under the following categories.
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/maui-ui-designs.md:12]: Remove Dashboard category

Comment (MuniappanSubramanian) [inline MAUI/UIKit/maui-ui-designs.md:12]: Check screen count and update

File: MAUI/UIKit/maui-ui-designs.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+documentation: ug
+---
+
+# Syncfusion MAUI UI Designs 
+
+The Essential UI Kit includes 101 screens which is under the following categories. 
+
+## Forms	
+
+* [Simple Login Page](https://github.com/syncfusion/essential-ui-kit-for-.net-maui/blob/master/EssentialUIKit/Views/Forms/SimpleLoginPage.xaml)
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/maui-ui-designs.md:16]: In the categories some screens are removed in MAUI check and remove

File: MAUI/visual-studio-integration/Essential-UI-Kit.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+   Title="Home"
+   ContentTemplate="{DataTemplate local:ItemName}"
+   Route="ItemName" />
+{% endhighlight %}
+
+Example: If you added Login With Social Icon Page,
+
+{% highlight xaml %}
+<Shell
+    . . .
```
Comment (MuniappanSubramanian) [inline MAUI/visual-studio-integration/Essential-UI-Kit.md:73]: Don t use shell content

File: MAUI/UIKit/GettingStarted.md (hunk 1 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md]: Changed the Sample Codes as per Net 8 Sample Code in Latest Commits.

File: MAUI/UIKit/maui-ui-designs.md (hunk 2 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/maui-ui-designs.md]: Removed the Dashboard Category in Latest Commits.

Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/maui-ui-designs.md]: Updated with Latest Commits

File: MAUI/UIKit/maui-ui-designs.md (hunk 3 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/maui-ui-designs.md]: Removed and Ensured in Latest Commits

File: MAUI/visual-studio-integration/Essential-UI-Kit.md (hunk 4 above)
Comment (HadayaRahman-SF4673) [reply MAUI/visual-studio-integration/Essential-UI-Kit.md]: Changed in Latest Commits

File: MAUI/UIKit/GettingStarted.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: Getting Started with MAUI UI Kit control | Syncfusion
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:3]: Check Title and Description all MD files

File: MAUI/UIKit/GettingStarted.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+### Steps to Use Essential UI Kit for MAUI Extension
+
+1. Open Visual Studio.
+
+2. Go to Extension, and then click Manage Extensions as shown in the following screenshot.
+
+   ![Visual Studio Extensions](UI-Kit-images/VS_Extensions.png)
+
+3. Search for **Essential UI Kit for MAUI**, and then install it.
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:31]: Name:
.NET MAUI Essential Ul Kit

File: MAUI/UIKit/GettingStarted.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+   ![Visual Studio Ui Kit Resources](UI-Kit-images/Essential_UIKit_ClassFiles.png)
+
+## How to render the added page
+
+In a MAUI demo application, to set the added page as the start-up page, you must define it in the App.xaml.cs file.
+For example, if you added the Login with Social Icon Page, you should invoke the page as shown in the following code:
+
+{% highlight xaml %}
+
+MainPage=new SampleFormsApplication.Views.Forms.LoginWithSocialIcon();
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:63]: MainPage=new LoginWithSocialIcon();

File: MAUI/UIKit/Overview.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+## Supported Platforms
+
+* Android 5.0 (API 21) or higher is required.
+* iOS 12.2 or higher is required.
+* macOS 12 or higher, using Mac Catalyst.
+* Windows 11 and Windows 10 version 1809 or higher
+
+
+To get started with Syncfusion XAML Pages for MAUI, refer to [Getting Started]()
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/Overview.md:43]: check needed?

File: MAUI/UIKit/maui-ui-designs.md [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+layout: post
+title: MAUI Essential UI Kit Designs | Syncfusion
+description: Essential UI Kit contains elegantly designed XAML templates for MAUI apps. These templates are compatible with Android, iOS, and UWP platforms.
+platform: maui
+control: MAUI UI Kit
+documentation: ug
+---
+
+# Syncfusion MAUI UI Designs 
+
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/maui-ui-designs.md:11]: Confirm categories and screen counts

File: MAUI/UIKit/GettingStarted.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+layout: post
+title: Getting Started with MAUI UI Kit control | Syncfusion
+description: Learn here about getting started with Syncfusion Essential Studio MAUI UI Kit control, its elements and more.
+platform: maui
+control: MAUI UI Kit
+documentation: ug
+---
+
+# Getting Started with MAUI UI Kit
+
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:11]: MAUI UI Kit - check naming

File: MAUI/UIKit/Overview.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+title: About MAUI UI Kit control | Syncfusion
+description: Learn here all about introduction of Syncfusion Essential Studio MAUI UI Kit control, its elements and more.
+platform: maui
+control: MAUI UI Kit
+documentation: ug
+---
+
+# MAUI UI Kit Overview
+
+The [Syncfusion Essential UI Kit]() is a collection of easy-to-use, extendable, and adaptable XAML pages designed to help you quickly build a MAUI application. It provides a set of generic, pre-defined screens that can be easily customized. Currently, the UI Kit includes screens for the following categories:
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/Overview.md:12]: The **.NET MAUI Essential Ul Kit** is a...... --> Change

Comment (MuniappanSubramanian) [inline MAUI/UIKit/Overview.md:12]: **Use this:**

The Syncfusion .NET MAUI Essential Ul Kit is a comprehensive collection of user-friendly, extendable, and adaptable XAML pages designed to accelerate the development of MAUI applications. It offers a range of pre-defined, generic screens that can be easily customized to meet your requirements. Currently, the UI Kit includes screens categorized under the following sections:

File: MAUI/UIKit/GettingStarted.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+The UI Kit screens can be added to your application in the following two ways:
+
+1. Using **Essential UI Kit for MAUI** Visual Studio extension.
+
+2. Copying the files from our open-source [GitHub repository]().
+
+
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:18]: There are two ways for including the UI Kit screens into your application:

* using the Visual Studio extension's .NET MAUI Essential Ul Kit.

* copying the files from our GitHub repository, which is open source.

File: MAUI/UIKit/GettingStarted.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+## Essential UI Kit for MAUI Extension
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:19]: Essential UI Kit for .NET MAUI Extension

File: MAUI/UIKit/GettingStarted.md [hunk 14]
```diff
@@ -0,0 +1,73 @@
...
+
+This is the simplest method to integrate pre-defined screens into your application. The following steps outline how to add screens to your app using our extension:
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:21]: This is the simplest way to integrate pre-defined screens into your application. Follow these steps to add screens to your app using our extension:

File: MAUI/UIKit/GettingStarted.md [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+
+### Steps to Use Essential UI Kit for MAUI Extension
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:23]: Remove this title

File: MAUI/UIKit/GettingStarted.md [hunk 16]
```diff
@@ -0,0 +1,73 @@
...
+
+3. Search for **Essential UI Kit for MAUI**, and then install it.
+
+   ![Visual Studio Extensions and Updates](UI-Kit-images/Extension_Update.png)
+
+4. Restart the Visual Studio and allow it to complete the installation. 
+
+5. Now, open an existing MAUI application or create a new application as per your requirements.
+ 
+6. Right-click the MAUI [NETStandard] project, and you can see the **Essential UI Kit for MAUI** option.
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:39]: Essential UI Kit for MAUI ---->.   .NET MAUI Essential Ul Kit

File: MAUI/UIKit/GettingStarted.md [hunk 17]
```diff
@@ -0,0 +1,73 @@
...
+
+   ![Visual Studio UIkit Category](UI-Kit-images/Essential_UIKit_PageName.png)
+
+9. Clicking the 'Add' button will include the selected page to your project. The necessary class files, resources, and NuGet package references will automatically be added to your project as shown in the following screenshot.
+
+   ![Visual Studio Ui Kit Resources](UI-Kit-images/Essential_UIKit_Resources.png)
+   ![Visual Studio Ui Kit Resources](UI-Kit-images/Essential_UIKit_Nuget_Packages.png)
+   ![Visual Studio Ui Kit Resources](UI-Kit-images/Essential_UIKit_ClassFiles.png)
+
+## How to render the added page
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:56]: How to Render the Added Page?

File: MAUI/UIKit/GettingStarted.md [hunk 18]
```diff
@@ -0,0 +1,73 @@
...
+
+In a MAUI demo application, to set the added page as the start-up page, you must define it in the App.xaml.cs file.
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/GettingStarted.md:58]: In a MAUI demo application, to set the added page as the startup page, you need to define it in the `App.xaml.cs` file. For instance, if you’ve added the **Login with Social Icon Page**, you can set it as the startup page using the following code snippet:


Note: please highlight key words

File: MAUI/UIKit/maui-ui-designs.md [hunk 19]
```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: MAUI Essential UI Kit Designs | Syncfusion
+description: Essential UI Kit contains elegantly designed XAML templates for MAUI apps. These templates are compatible with Android, iOS, and UWP platforms.
+platform: maui
+control: MAUI UI Kit
+documentation: ug
+---
+
+# Syncfusion MAUI UI Designs
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/maui-ui-designs.md:10]: Syncfusion .NET MAUI UI Designs

File: MAUI/UIKit/maui-ui-designs.md [hunk 20]
```diff
@@ -0,0 +1,154 @@
...
+title: MAUI Essential UI Kit Designs | Syncfusion
+description: Essential UI Kit contains elegantly designed XAML templates for MAUI apps. These templates are compatible with Android, iOS, and UWP platforms.
+platform: maui
+control: MAUI UI Kit
+documentation: ug
+---
+
+# Syncfusion MAUI UI Designs 
+
+The Essential UI Kit offers a collection of 87 screens, organized into 18 versatile categories, designed to streamline your design process and elevate your user interface creations.
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/maui-ui-designs.md:12]: check naming

File: MAUI/UIKit/page-and-template-dependencies.md [hunk 21]
```diff
@@ -0,0 +1,1574 @@
+---
+layout: post
+title:  XAML Templates | Page Template Dependencies of UI Kit | Syncfusion
+description: The list of screen-wise resources and NuGet packages required for each Individual MAUI Essential UI kit Screens.
+platform: maui
+control: UI kit
+documentation: ug
+---
+
+This page list the dependencies ( converters, rendereres and NuGet packages) of each XAML page in Essential UI Kit project
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/page-and-template-dependencies.md:10]: This page lists the dependencies (converters, renderers, and NuGet packages) required for each XAML page in the Essential UI Kit project.

File: MAUI/UIKit/page-and-template-dependencies.md [hunk 22]
```diff
@@ -0,0 +1,1574 @@
...
+title:  XAML Templates | Page Template Dependencies of UI Kit | Syncfusion
+description: The list of screen-wise resources and NuGet packages required for each Individual MAUI Essential UI kit Screens.
+platform: maui
+control: UI kit
+documentation: ug
+---
+
+This page list the dependencies ( converters, rendereres and NuGet packages) of each XAML page in Essential UI Kit project  
+
+# Page Template Dependencies of MAUI UI Kit
```
Comment (MuniappanSubramanian) [inline MAUI/UIKit/page-and-template-dependencies.md:12]: # Page Template Dependencies of .NET MAUI UI Kit

File: maui-toc.html [hunk 23]
```diff
@@ -149,6 +153,16 @@
             </li>
         </ul>
     </li>
+    <li>
+        UI Kit
+        <ul>
+            <li><a href="/maui/UIKit/Overview">Overview</a></li>
+            <li><a href="/maui/UIKit/GettingStarted">Getting Started</a></li>
+            <li><a href="/maui/UIKit/maui-ui-designs">MAUI Designs</a></li>
```
Comment (MuniappanSubramanian) [inline maui-toc.html:161]: MAUI UI Designs

File: maui-toc.html [hunk 24]
```diff
@@ -120,9 +120,13 @@
             <li>
                 <a href="/maui/visual-studio-integration/Toolbox-Control">Toolbox Control</a>
             </li>
+            <li>
+                <a href="/maui/visual-studio-integration/Essential-UI-Kit">Essential UI Kit</a>
```
Comment (MuniappanSubramanian) [inline maui-toc.html:124]: .NET MAUI Essential UI Kit

File: MAUI/UIKit/GettingStarted.md (hunk 5 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md:3]: Checked and Updated in Latest Commit

File: MAUI/UIKit/GettingStarted.md (hunk 6 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md:31]: Changed Respective Names in Latest Commits

File: MAUI/UIKit/GettingStarted.md (hunk 7 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md:63]: Changed the Format on Latest Commits

File: MAUI/UIKit/Overview.md (hunk 8 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/Overview.md:43]: Removed in Latest Commits

File: MAUI/visual-studio-integration/Essential-UI-Kit.md [hunk 25]
```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: Essential UI Kit | MAUI | Syncfusion
+description: The Syncfusion MAUI Essential UI Kit extension provides the predefined design Screens for the MAUI Apps.
+platform: maui
+control: Syncfusion Extensions
+documentation: ug
+---
+
+# MAUI Essential UI Kit
```
[the remark this answers is not in the document]
Comment (HadayaRahman-SF4673) [reply MAUI/visual-studio-integration/Essential-UI-Kit.md:10]: Checked and Changed It

File: MAUI/visual-studio-integration/Essential-UI-Kit.md [hunk 26]
```diff
@@ -0,0 +1,67 @@
...
+
+7.	Then, Syncfusion licensing registration required message box will be shown if you installed the trial setup or NuGet packages since Syncfusion introduced the licensing system from 2018 Volume 2 (v16.2.0.41) Essential Studio release. Navigate to the [help topic](https://help.syncfusion.com/common/essential-studio/licensing/overview#how-to-generate-syncfusion-license-key), which is shown in the licensing message box to generate and register the Syncfusion license key to your project. Refer to this [blog](https://www.syncfusion.com/blogs/post/whats-new-in-2018-volume-2.aspx) post for understanding the licensing changes introduced in Essential Studio. 
+
+## Run the UI template item
+
+To set your preferred UI Template as the start page of your application, simply open the App.xaml.cs file in your MAUI project and make the following changes.
+
+{% highlight xaml %}
+
+MainPage=new SampleFormsApplication.Views.CategoryName.PageName();
```
[the remark this answers is not in the document]
Comment (HadayaRahman-SF4673) [reply MAUI/visual-studio-integration/Essential-UI-Kit.md:57]: Changed on latest commit

File: maui-toc.html (hunk 24 above)
Comment (HadayaRahman-SF4673) [reply maui-toc.html]: Changed In Latest Commits

File: maui-toc.html (hunk 23 above)
Comment (HadayaRahman-SF4673) [reply maui-toc.html]: Changed In Latest Commits

File: MAUI/UIKit/page-and-template-dependencies.md (hunk 22 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/page-and-template-dependencies.md]: Changed In Latest Commits

File: MAUI/UIKit/page-and-template-dependencies.md (hunk 21 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/page-and-template-dependencies.md]: Changed In Latest Commits

File: MAUI/UIKit/maui-ui-designs.md (hunk 20 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/maui-ui-designs.md]: Changed In Latest Commits

File: MAUI/UIKit/maui-ui-designs.md (hunk 19 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/maui-ui-designs.md]: Changed In Latest Commits

File: MAUI/UIKit/GettingStarted.md (hunk 18 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md]: Changed In Latest Commits

File: MAUI/UIKit/GettingStarted.md (hunk 17 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md]: Changed In Latest Commits

File: MAUI/UIKit/GettingStarted.md (hunk 16 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md]: Changed In Latest Commits

File: MAUI/UIKit/GettingStarted.md (hunk 15 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md]: Changed In Latest Commits

File: MAUI/UIKit/GettingStarted.md (hunk 14 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md]: Changed In Latest Commits

File: MAUI/UIKit/GettingStarted.md (hunk 13 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md]: Changed In Latest Commits

File: MAUI/UIKit/GettingStarted.md (hunk 12 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md:18]: Changed In Latest Commits

File: MAUI/UIKit/Overview.md (hunk 11 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/Overview.md]: Changed In Latest Commits

File: MAUI/UIKit/GettingStarted.md (hunk 10 above)
Comment (HadayaRahman-SF4673) [reply MAUI/UIKit/GettingStarted.md:11]: Changed In Latest Commits

**Review by MuniappanSubramanian — APPROVED** [review]

**Pull request merged**